### DISCUSSION TITLE
Recommended models for extremely poor quality source audio?

### DISCUSSION THREAD

**Discussion by Ottonis777** [Anjok07/ultimatevocalremovergui#1325] [Q&A]
First of all, let me preface how deeply grateful I am for all the work the developers have put into UVR5. It easily outperformes quite a few commercially available tools that I have tried in the past, such as RX10 or SpectraLayers, so this alone is already a feat worth praising. The GUI is also very well designed and integrates  maximal functionality with fantastic legibility and ease of use. 

Now my question:
I am working on restoring/restaurating very old audio material, mostly from old cassettes/tapes with audio recordings being between 30-50 years old. The audio signal is not only fairly degraded but it has also been recorded very poorly: poor dynamic range, poor signal-to noise ratio, and different instruments and vocals being a "muddy mess" with lack of bass and lack of high frequencies.
I have tried a few models, including the new MDX23C HQ2 as well as a few combinations of models in ensemble mode, but all these models seem to struggle with this type of audio material.

Are there any models (or combination thereof) currently available that in your experience might be better suitable for separating stems from extremely poor audio sources? I would be happ to pay if the developers could train their AI for such type of material, and I am certainly not alone, since audio restauration has become quite a growing niche lately,

Thanks in advance for your experiences and your help.

Best
Otto

**Comment by magicaltrevor1970** [comment]
I too would be very interested in this. I have some cassette recordings from 1991 of a band I was in. The lead singer passed shortly after, and the rest of us would love to restore these recordings. Any information in this regard would be most welcome.
